applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -927,7 +940,13 @@ static void report_issued(const void *subscriber_id, uint8_t report_id, bool err
 		if ((next_rs->state != STATE_DISCONNECTED) &&
 		    (next_rs->linked_rd->linked_rs == next_rs) &&
 		    (next_rs->cnt == 0)) {
-			if (report_send(next_rs->linked_rd, false, false)) {
+			if (report_send(next_rs->linked_rd, next_rs, false, false)) {
+				break;
+			}
+		}
+
+		if (next_rs->clearing_needed) {
```

Comment:
this can be removed

Reply:
empty_rd is already linked to this rs and rs->update_needed is set to true so if clearing was not already sent out it will be